identity.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/iancoleman/orderedmap"
 	"golang.org/x/crypto/blake2b"
 
-	"golang.org/x/crypto/ed25519"
+	"crypto/ed25519"
```

Comment:
Nitpick: could be regrouped with imports above.

Reply:
`goimports`ed